frontend/components/TableContainer/DataTable/SoftwareNameCell/SoftwareNameCell.tsx

```diff
@@ -12,6 +12,18 @@ import TooltipTruncatedTextCell from "../TooltipTruncatedTextCell";
 
 const baseClass = "software-name-cell";
 
+/**
+ * Normalizes software names for display purposes.
+ * Maps known problematic names to their user-friendly display names.
+ */
+const normalizeSoftwareDisplayName = (name: string): string => {
+  // Handle cases where name might have version appended (e.g., "Microsoft.CompanyPortal, 11.2.1495.0")
+  if (name.startsWith("Microsoft.CompanyPortal")) {
+    return name.replace("Microsoft.CompanyPortal", "Company Portal");
+  }
+  return name;
+};
+
```

Comment:
One issue, I can see is if someone searches "Company Portal", it will come up empty because it uses the API which has it stored as "Microsoft.CompanyPortal".

Food for thought on next iteration to change this to a BE change for `display_name` and get the search by `display_name` running in the BE.